genes-brains-and-behavior.csl

```diff
@@ -0,0 +1,231 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="en-US">
+  <info>
+    <title>Genes, Brain and Behavior</title>
+    <id>http://www.zotero.org/styles/genes-brains-and-behavior</id>
+    <link href="http://www.zotero.org/styles/genes-brains-and-behavior" rel="self"/>
+    <link href="http://www.zotero.org/styles/journal-of-zoology" rel="template"/>
+    <link href="http://onlinelibrary.wiley.com/journal/10.1111/(ISSN)1601-183X/homepage/ForAuthors.html" rel="documentation"/>
+    <author>
+      <name>Patrick O'Brien</name>
+    </author>
+    <category citation-format="author-date"/>
+    <category field="biology"/>
+    <category field="psychology"/>
+    <issn>1601-1848</issn>
+    <eissn>1601-183X</eissn>
+    <updated>2017-08-31T16:33:56+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <macro name="container">
+    <choose>
+      <if type="chapter paper-conference" match="any">
+        <text term="in" text-case="capitalize-first" suffix=" "/>
+        <names variable="editor translator" delimiter=", " suffix=", ">
+          <name name-as-sort-order="all" and="symbol" sort-separator=", " initialize-with="." delimiter=", " delimiter-precedes-last="never"/>
+          <label form="short" strip-periods="true" prefix=" (" suffix=")"/>
+        </names>
+        <text variable="container-title" font-style="italic"/>
+        <text variable="collection-title" prefix=", "/>
+      </if>
+      <else>
+        <text variable="container-title" form="short" strip-periods="true" font-style="italic"/>
+        <text variable="collection-title" prefix=", "/>
+      </else>
+    </choose>
+  </macro>
+  <macro name="author">
+    <names variable="author">
+      <name name-as-sort-order="all" and="symbol" sort-separator=", " initialize-with="." delimiter=", " delimiter-precedes-last="never"/>
+      <label form="short" prefix=" (" suffix=")" text-case="capitalize-first"/>
```

Comment:
probably hard to find an example, but unless you have one to the contrary, make this lowercase & strip periods just like for editors in chapter citations.